From 98ecf7d2c5753357fd2c1d1e8e72f425e4d48f9b Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Thu, 12 Dec 2013 15:50:27 +0100
Subject: [PATCH] snmpstats: fixe MIB file definitions

- patch by takeshi, FS#363

(cherry picked from commit ab75785aa707f10fdbccfb0531a0f35e820f5ebc)
---
 modules/snmpstats/mibs/KAMAILIO-TC | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/modules/snmpstats/mibs/KAMAILIO-TC b/modules/snmpstats/mibs/KAMAILIO-TC
index 81d1459..171c491 100644
--- a/modules/snmpstats/mibs/KAMAILIO-TC
+++ b/modules/snmpstats/mibs/KAMAILIO-TC
@@ -62,7 +62,7 @@ KamailioSIPTransportProtocol ::= TEXTUAL-CONVENTION
                             tcp(2),
                             sctp(3),
                             tls(4),
-                            sctp_tls(5),
+                            sctptls(5),
 			    ws(6),
 			    wss(7)
            }
@@ -79,14 +79,13 @@ KamailioSIPEntityRole ::= TEXTUAL-CONVENTION
 		operation of a SIP outbound edge proxy node.
 		Kamailio adds the role 'sipcaptureServer' for
 		operation of a Homer SIPcapture node."
-		"
            SYNTAX BITS {
                             other(0),
                             userAgent(1),
                             proxyServer(2),
                             redirectServer(3),
-                            registrarServer(4)
-                            edgeproxyServer(5)
+                            registrarServer(4),
+                            edgeproxyServer(5),
                             sipcaptureServer(6)
            }
 
-- 
1.8.5.2

